train.py

```diff
@@ -57,45 +57,22 @@ def _update_networks(args, T, model, shared_model, shared_average_model, loss, o
   for shared_param, shared_average_param in zip(shared_model.parameters(), shared_average_model.parameters()):
     shared_average_param = args.trust_region_decay * shared_average_param + (1 - args.trust_region_decay) * shared_param
 
-
-# Enables/disables gradients of model apart from policy head
-def _isolate_policy_grads(model, isolate):
-  for name, param in model.named_parameters():
-    if 'fc_actor' not in name:
-      param.requires_grad = not isolate
-
-
 # Computes an "efficient trust region" loss (policy head only) based on an existing loss and two distributions
-def _trust_region_loss(model, distribution, ref_distribution, loss, threshold):
-  _isolate_policy_grads(model, True)  # Disable gradients for other parameters
-  # Compute gradients from original loss
-  model.zero_grad()
-  loss.backward(retain_graph=True)
-  # Gradients should be treated as constants (not using detach as volatility can creep in when double backprop is not implemented)
-  g = [model.fc_actor.weight.grad.data.clone(), model.fc_actor.bias.grad.data]
-  model.zero_grad()
-
-  # KL divergence k ← ∇θ0∙DKL[π(∙|s_i; θ_a) || π(∙|s_i; θ)]
-  kl = (ref_distribution * (ref_distribution.log() - distribution.log())).sum(1).mean(0)
-  # Compute gradients from (negative) KL loss (increases KL divergence)
-  (-kl).backward(retain_graph=True)
-  k = [model.fc_actor.weight.grad.data.clone(), model.fc_actor.bias.grad.data]
-  model.zero_grad()
+def _trust_region_loss(model, distribution, ref_distribution, loss, threshold, g, k):
+
+  kl = -(ref_distribution*(distribution.log()-ref_distribution.log())).sum(1).mean(0)
```

Comment:
Please put spaces between maths operators and variables (`*` and `-` here). Same on line 74.